assets/js/frontend-form.js

```diff
@@ -933,18 +942,21 @@
                 }
             },
 
-            textLimit: function(event, limit, limit_type) {
+            textLimit: function(event, limit, limit_type, limit_to ) {
                 var self = $(this),
                     content_length = self.val().length + 1;
 
                     if ( 'word' === limit_type ) {
                         content_length = self.val().split(' ').length;
                     }
 
-                if ( limit && content_length > limit ) {
-                    self.closest('.wpuf-fields').find('span.wpuf-wordlimit-message').html( WP_User_Frontend.content_limit_message( limit_type ) );
+                if ( limit && content_length > limit && 'max' === limit_to ) {
+                    WP_User_Frontend.content_limit_message( self, limit_type, limit_to, limit )
                     WP_User_Frontend.editorLimit.blockTyping(event);
-                } else {
+                }else if ( limit && content_length < limit && 'min' === limit_to ) {
```

Comment:
You've problems with space in the `else` parenthesis I guess, I saw it multiple times. Everywhere I see the spacing problem. Fix this one and the line below, and make sure it doesn't happen in the future.